__tests__/app.test.js

```diff
@@ -213,3 +213,53 @@ describe("/api/articles/:article_id/comments", () => {
       });
   });
 });
+describe("/api/articles/:article_id", () => {
```

Comment:
There are a couple more tests we could have here:

What if the request has the correct key but the data is invalid eg `{ inc_votes: "one" }`?

In a similar vein what if the id of the article in the url is invalid eg a string?